azure-monitor-opentelemetry/azure/monitor/opentelemetry/util/configurations.py

```diff
@@ -15,7 +61,121 @@ def _get_configurations(**kwargs) -> Dict[str, ConfigurationValue]:
     for key, val in kwargs.items():
         configurations[key] = val
 
+    _default_exclude_instrumentations(configurations)
+    _default_disable_logging(configurations)
+    _default_disable_metrics(configurations)
+    _default_disable_tracing(configurations)
+    _default_logging_level(configurations)
+    _default_logger_name(configurations)
+    _default_logging_export_interval_ms(configurations)
+    _default_metric_readers(configurations)
+    _default_views(configurations)
+    _default_sampling_ratio(configurations)
+    _default_tracing_export_interval_ms(configurations)
+    _default_instrumentation_config(configurations)
+
+    # TODO: remove when validation added to BLRP
+    if configurations[LOGGING_EXPORT_INTERVAL_MS_ARG] <= 0:
+        raise ValueError(
+            "%s must be positive." % LOGGING_EXPORT_INTERVAL_MS_ARG
+        )
+
     return configurations
 
 
-# TODO: Add env var configuration
+def _default_exclude_instrumentations(configurations):
+    if EXCLUDE_INSTRUMENTATIONS_ARG not in configurations:
+        default = []
+        if EXCLUDE_INSTRUMENTATIONS_ENV_VAR in environ:
+            default = loads(environ[EXCLUDE_INSTRUMENTATIONS_ENV_VAR])
+        configurations[EXCLUDE_INSTRUMENTATIONS_ARG] = default
+
+
+def _default_disable_logging(configurations):
+    if DISABLE_LOGGING_ARG not in configurations:
+        default = False
+        if DISABLE_LOGGING_ENV_VAR in environ:
+            default = bool(environ[DISABLE_LOGGING_ENV_VAR])
+        configurations[DISABLE_LOGGING_ARG] = default
+
+
+def _default_disable_metrics(configurations):
+    if DISABLE_METRICS_ARG not in configurations:
+        default = False
+        if DISABLE_METRICS_ENV_VAR in environ:
+            default = bool(environ[DISABLE_METRICS_ENV_VAR])
+        configurations[DISABLE_METRICS_ARG] = default
+
+
+def _default_disable_tracing(configurations):
+    if DISABLE_TRACING_ARG not in configurations:
+        default = False
+        if DISABLE_TRACING_ENV_VAR in environ:
+            default = bool(environ[DISABLE_TRACING_ENV_VAR])
+        configurations[DISABLE_TRACING_ARG] = default
+
+
+def _default_logging_level(configurations):
+    if LOGGING_LEVEL_ARG not in configurations:
+        default = NOTSET
+        if LOGGING_LEVEL_ENV_VAR in environ:
+            default = int(environ[LOGGING_LEVEL_ENV_VAR])
```

Comment:
We should probably have error handling for these casts.

Reply:
Done